packages/react-components/src/components/VideoGallery/useVideoTileContextualMenuProps.ts

```diff
@@ -209,25 +302,59 @@ export const useVideoTileContextualMenuProps = (props: {
 
     return { items, styles: {}, calloutProps: { preventDismissOnEvent }, shouldFocusOnContainer: false };
   }, [
-    scalingMode,
-    strings,
-    view,
-    isPinned,
-    onPinParticipant,
-    onUnpinParticipant,
-    onUpdateScalingMode,
+    onMuteParticipant,
+    strings?.muteParticipantMenuItemLabel,
+    strings?.unpinParticipantForMe,
+    strings?.pinParticipantForMe,
+    strings?.unpinParticipantMenuItemAriaLabel,
+    strings?.pinnedParticipantAnnouncementAriaLabel,
+    strings?.pinParticipantForMeLimitReached,
+    strings?.stopSpotlightOnSelfVideoTileMenuLabel,
+    strings?.stopSpotlightVideoTileMenuLabel,
+    strings?.addSpotlightVideoTileMenuLabel,
+    strings?.startSpotlightVideoTileMenuLabel,
+    strings?.spotlightLimitReachedMenuTitle,
+    strings?.fitRemoteParticipantToFrame,
+    strings?.fillRemoteParticipantFrame,
+    participant.isMuted,
     participant.userId,
     participant.displayName,
-    disablePinMenuItem,
-    toggleAnnouncerString,
-    spotlightedParticipantUserIds,
+    isPinned,
     isSpotlighted,
-    onStartSpotlight,
+    scalingMode,
+    onUnpinParticipant,
+    onPinParticipant,
+    toggleAnnouncerString,
+    disablePinMenuItem,
+    myUserId,
     onStopSpotlight,
+    spotlightedParticipantUserIds,
     maxParticipantsToSpotlight,
-    myUserId,
-    onMuteParticipant,
-    participant.isMuted
+    onStartSpotlight,
+    onUpdateScalingMode,
+    view,
+    /* @conditional-compile-remove(media-access) */
+    strings?.permitAudioTileMenuLabel,
```

Comment:
Let's say there is a strings.abc. If the value strings.abc changes it will cause this component to render although string.abc is not in use here. All dependencies are auto generated, let's not change it manually.

Reply:
It was strings in the dependency, will revert back to it was.

Reply:
Done